test/blackbox-tests/test-cases/dune-preprocess.t/run.t

```diff
@@ -0,0 +1,9 @@
+This test check that we can show the preprocessed output of a source code
```

Comment:
```suggestion
This test check that we can show the preprocessed output of a source code
```
Should this be "This test check is so we can show" ... or "This test check shows the preprocessed..."

Reply:
Oops, I meant `s/check/checks/g`...!

I'm willing to fix this but this would outdate this thread: https://github.com/ocaml/dune/pull/5615#pullrequestreview-952113916

Can I hold this until we decide the name of the command?

Reply:
Absolutely!